sdk/servicebus/azure-messaging-servicebus/CHANGELOG.md

```diff
@@ -28,6 +27,12 @@
 - Upgraded `azure-core` dependency to `1.11.0`.
 - Upgraded `azure-core-amqp` dependency to `2.0.0`.
 
+### Known issues
+- ClassNotFoundException for BinaryData : When user add 'azure-identity' and 'azure-messaging-servicebus' as 
```

Comment:
**Unhandled ClassNotFoundException exception when using `azure-messaging-servicebus` 7.0.0 and other Azure SDKs in the same pom.xml file.**

**Issue**

This exception occurs due to a known `azure-core:1.11.0` dependency conflict between `azure-messaging-servicebus:7.0.0` and other Azure SDKs, such as `azure-identity:1.2.0`, which depend on older versions of `azure-core`.  

When two libraries depend on different versions of the same package, Maven will simply pick the first version that it sees and ignores the others.

**Resolution**

A workaround is to change the order of dependencies in the application's pom.xml file to list `azure-messaging-servicebus` before any other `azure-*` client libraries.

A Fix for the January 2021 release is being tracked **_HERE_**

**Additional Information**

More information on diagnosing and resolving dependency conflicts can be found here: https://github.com/Azure/azure-sdk-for-java/wiki/Frequently-Asked-Questions#dependency-conflict

Reply:
also should this go into a known-issues file that's separate from the changelog or should these just be included in the changelog.md?

/cc @samvaity

Reply:
As per our last guidance, this should go as a separate known-issue and we could do a mention in the changelog.
@czubair Could you confirm the correct way of addressing this?

Reply:
Yes, that is correct @samvaity

Reply:
I have added separate file for known-issues.